.storybook/preview.tsx

```diff
@@ -32,9 +33,11 @@ const preview: Preview = {
       <SiteProvider>
         <LocaleProvider>
           <NotificationProvider viewportClassName="absolute top-0 left-0 list-none">
-            <div className={rootClasses}>
-              <Story />
-            </div>
+            <ThemeProvider>
```

Comment:
We shouldn't use `ThemeProvider` on Storybook. We intentionally want the Theme Change to be done on Storybook Controls (as you can see on the "Dark / Light" theme chooser.

Hence, to make the Component happy because you're using `useTheme` you could probably either mock the ThemeProvider or something else, since by using the ThemeProvider the component/story becomes flaky (as ThemeProvider (next-themes)) injects some stuff on the HTML element.

Reply:
I could only find a solution of decorating the story at some level with the provider. In the storybook, the component does not yell when it's not wrapped with the `ThemeProvider` except that the theme-toggle-button can't do its job without a provider. Using the Storybook theme control is working for now using your suggestion here
> I recommend doing what we did here: https://github.com/nodejs/nodejs.org/blob/main/components/Common/ThemeToggle/index.tsx#L25-L26

Reply:
Right, let's not use the `useTheme` then for this Component as mentioned on below discussions. So that the Component is pretty much agnostic (as the LanguageDropdown and ThemeToggle) components are.